### PR TITLE
All: Add quilt loader support :)

### PR DESCRIPTION
**Pull request by kristofbolyai**
Works fine in production and development environment.

### CODE REVIEW

**Comment by kristofbolyai** [comment]
@DevScyu Can you fix the build task? (and also prepare jenkins)

EDIT: build task fixed itself lol. Jenkins still needs preparation

EDIT 2: build task is not fixed.

EDIT 3: build task is now fixed.

**Comment by magicus** [comment]
Do we really want to support Quilt? Here we were, happy by having "just" two mod loaders, and then the Quilt team comes along, breeding strife and discord within the modding community. (Mostly joking, but partially serious... I'm not enjoying seeing Yet Another Modloader gaining foothold.)

**Comment by kristofbolyai** [comment]
> Do we really want to support Quilt? Here we were, happy by having "just" two mod loaders, and then the Quilt team comes along, breeding strife and discord within the modding community. (Mostly joking, but partially serious... I'm not enjoying seeing Yet Another Modloader gaining foothold.)

I mean it is mostly architectury that does the support, and if we use it, we might as well support Quilt. If something comes up, we can remove the support.

File: fabric/src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   ],
   "contact": {
     "homepage": "https://wynntils.com",
-    "sources": "https://github.com/magicus/WynnArchi"
+    "sources": "https://github.com/Wynntils/Artemis"
```
Comment (magicus) [inline fabric/src/main/resources/fabric.mod.json:7]: Oops! :-)

File: forge/build.gradle [hunk 2]
```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "com.github.johnrengelman.shadow" version "7.1.0"
+    id "com.github.johnrengelman.shadow" version "7.1.2"
```
Comment (magicus) [inline forge/build.gradle:2]: Maybe move the version into gradle.properties instead of duplicating it across mod loaders?

**Review by magicus — DISMISSED** [review]
Looks okay. Maybe fix the shadow plugin version...

**Comment by kristofbolyai** [comment]
You cannot use variables for plugin version.

**Comment by magicus** [comment]
Ah. Too bad. (What a weird restriction)

**Review by magicus — APPROVED** [review]

**Pull request merged**